README.md

```diff
@@ -82,16 +82,16 @@ require 'vendor/autoload.php';
 // If you are not using Composer
 // require("path/to/sendgrid-php/sendgrid-php.php");
 
-$from = new SendGrid\Email("Example User", "test@example.com");
+$from = new \SendGrid\Mail\Email("Example User", "test@example.com");
 $subject = "Sending with SendGrid is Fun";
-$to = new SendGrid\Email("Example User", "test@example.com");
-$content = new SendGrid\Content("text/plain", "and easy to do anywhere, even with PHP");
-$mail = new SendGrid\Mail($from, $subject, $to, $content);
+$to = new \SendGrid\Mail\Email("Example User", "test@example.com");
+$content = new \SendGrid\Mail\Content("text/plain", "and easy to do anywhere, even with PHP");
+$msg = new \SendGrid\Mail\SendGridMessage($from, $subject, $to, $content);
 
 $apiKey = getenv('SENDGRID_API_KEY');
-$sg = new \SendGrid($apiKey);
+$sg = new \SendGrid\Factory($apiKey);
```

Comment:
I like your reasoning and I think this is a sane choice. And good catch on the Client naming clash! That said, I'm leaning towards a more descriptive "ClientFactory".